app/frontend/src/components/ContributorForm/ContributorForm.test.tsx

```diff
@@ -82,19 +83,18 @@ describe("contributor form", () => {
   });
 
   it("it shows the form again if processing the form fails", async () => {
```

Comment:
```suggestion
  it("shows the form again if processing the form fails", async () => {
```
Nitpick, but it bothers me to read `it` twice in the sentence 😅